app/src/commonMain/kotlin/Platform.kt

```diff
@@ -1,6 +1,6 @@
 /*
- * ONI Seed Browser
- * Copyright (C) 2025 Stefan Oltmann
+ * Oxygen Not Included Seed Browser
+ * Copyright (C) 2025 The Maps Not Included Authors
```

Comment:
The name should stay "ONI Seed Browser".

Instead of the AUTHORS file just add a copyright line to every file you touched / created if you like.